Phonebook.Source.MockPeopleBackend/index.js

```diff
@@ -86,7 +86,7 @@ function generatePerson() {
       ReceptionPhone: null,
       LinkPicture: null,
       LinkRoutingInfo: null,
-      City: { Name: room.Place, Building: room.Building },
+      City: { Name: room.Place, Building: room.Building, zipCode: faker.address.zipCode() },
```

Comment:
Should be done in another issue you opened #380.
So can be removed

Reply:
But it is ok for me, so I'm givin my go :)